fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -141,6 +146,10 @@
 
     protected abstract String externalPath();
 
+    protected Deskolemizer deskolemizer() {
+        return deskolemizer == null ? deskolemizer = new Deskolemizer(translator(), null) : deskolemizer;
```

Comment:
You should make such style considerations clear and get them into Checkstyle rules, if you intend to enforce them.